### PR TITLE
feat($structure, $stylesheet, $schema): Add Bonus section

### PR DESCRIPTION
**Pull request by j0ntsang**

### CODE REVIEW

File: src/sections/bonus.liquid [hunk 1]
```diff
@@ -1,24 +1,28 @@
-<section class="bonus {% if section.settings.background_color_preset != 'none' and section.settings.background_color_preset != blank %}{{ section.settings.background_color_preset | append: '-section-color-preset' }}{% endif %}" data-preview-item="bonus">
+<section class="bonus large-section {% if section.settings.background_color_preset != 'none' and section.settings.background_color_preset != blank %}{{ section.settings.background_color_preset | append: '-section-color-preset' }}{% endif %}" data-preview-item="bonus">
```
Comment (derek-findthebrink) [inline src/sections/bonus.liquid:1]: same here as with the other PR re: the classname

File: src/styles/_section_bonus.scss [hunk 2]
```diff
@@ -3,11 +3,52 @@
 
 .bonus {
 
-  .section__heading {}
+  .section__headings {
+    @include screen-lg-min {
+      @include make-col(4);
```
Comment (derek-findthebrink) [inline src/styles/_section_bonus.scss:8]: nice!

**Review by derek-findthebrink — APPROVED** [review]
I'm curious to see how far you wanna go with the grid system!!

File: src/sections/bonus.liquid (hunk 1 above)
Comment (j0ntsang) [reply src/sections/bonus.liquid:1]: Addressed in: https://github.com/thinkific/fundamentals/pull/4

**Pull request merged**